test/test_item.rb

```diff
@@ -123,6 +123,18 @@ def test_add_token_create
     refute_empty(add_token_response.expiration)
   end
 
+  def test_add_token_create_with_user_fields
+    add_token_response = client.item.add_token.create(
+      client_user_id: '123-fake-user-id',
+      legal_name: 'John Doe',
+      phone_number: '+1 415 555 0123',
+      phone_number_verified_time: '2020-01-01T00:00:00Z',
+      email_address: 'example@plaid.com'
```

Comment:
Should we also add `email_address_verified_time` to this test?

Reply:
⚡